modules/preview-react/_examples/stories/mdx/examples/Density.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {FormField, FormFieldGroup} from '@workday/canvas-kit-preview-react/form-field';
+import {FormField, FormFieldGroup} from '@workday/canvas-kit-react/form-field';
```

Comment:
I should move these stories out of preview after the promotion. Also reference this example in the upgrade guide

Reply:
done